node.js

```diff
@@ -18,11 +18,16 @@ module.exports = {
     es6: true,
     node: true,
   },
+  globals: {
+    __DEV__: true,
```

Comment:
This is a react-native specific global, it shouldn't belong here. Instead of defining the accepted globals I would define the ones that are not accepted by using this module:  https://github.com/sidoshi/eslint-restricted-globals

Reply:
Makes sense. We are already using that module too